.gitignore

```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+.idea/
```

Comment:
https://stackoverflow.com/questions/7335420/global-git-ignore

Remove this change

Reply:
531e7e6](https://github.com/Vizzuality/soils-revealed-lambda/pull/14/commits/531e7e6390f35c76effe82f928443746032e22d2)